README.md

```diff
@@ -316,6 +316,71 @@ Library specific options:
     ZSTD_INCLUDE_DIR                - Directory that contains zstd.h
     ZSTD_LIBRARY                    - Path to libzstd.a/libzstd.so/ztd.lib
 
+### Compiling for Windows using MinGW on GNU/Linux
+
+It is highly recommeded to compile on Ubuntu 18.04 or newer.
+
+#### Dependencies
+
+Install Ubuntu packages:
+
+    sudo apt install wget xz-utils unzip git cmake gettext
+
+Download the MinGW toolchain for 64-bit Windows and extract to `/usr`:
+
+    wget -nv http://minetest.kitsunemimi.pw/mingw-w64-i686_9.2.0_ubuntu18.04.tar.xz -O mingw-w32.tar.xz
+    sudo tar -xaf mingw-w32.tar.xz -C /usr
+
+Download the MinGW toolchain for 64-bit Windows and extract to `/usr`:
+
+    wget -nv http://minetest.kitsunemimi.pw/mingw-w64-x86_64_9.2.0_ubuntu18.04.tar.xz -O mingw-w64.tar.xz
+    sudo tar -xaf mingw-w64.tar.xz -C /usr
```

Comment:
Aside from installing to `/usr` not being a good idea I think this should just suggest people to install
`g++-mingw-w64-i686` and `g++-mingw-w64-x86-64` from apt.

Reply:
Good idea! However we need a way to deliver `libgcc_*.dll`, `libstdc++-*.dll`, `libwinpthread-*.dll`, as these DLLs will be missing when using the MinGW packages from apt.

Maybe we can do something like this to extract the extra DLLs from your MinGW archive to `/usr/x86_64-w64-mingw32/bin`:
```bash
tar -xaf mingw-w64.tar.xz x86_64-w64-mingw32/bin/*.dll -C /usr
```
Note this still requires extracting stuff to `/usr`, which is not ideal.

Instead, I can add a environment variable (like `EXTRA_DLLS_DIR`) that points to a directory with extra DLLs to the build scripts, so that the scripts include DLLs from there. Let me know what you think!

Reply:
> as these DLLs will be missing when using the MinGW packages from apt.

Will they? Ubuntu should have them and the script should be picking them up fine (after #11764 that is)

Reply:
Ubuntu's MinGW packages sure do contain these DLLs, but they're not detected correctly. See comment in #11764.

Reply:
See #11759 for a fix for the DLL detection issue